src/api/todos.ts

```diff
@@ -5,4 +5,21 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
+export const postTodos = async (todo: Omit<Todo, 'id'>) => {
+  const createdTodo = await client.post<Todo>('/todos', todo);
+
+  return createdTodo;
+};
+
+export const patchTodos = async (todo: Partial<Todo>, todoId: number) => {
```

Comment:
```suggestion
export const editTodo = async (todo: Partial<Todo>, todoId: number) => {
```
you edit only a single todo here